src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -139,41 +152,59 @@ <h5><span jhiTranslate="artemisApp.studentExams.participationStatus">Participati
                     {{ (passed ? 'global.generic.yes' : 'global.generic.no') | artemisTranslate }}
                 </li>
             </ol>
+
+            <button
+                class="btn btn-danger"
+                type="button"
+                *ngIf="course?.isAtLeastInstructor"
+                (click)="openConfirmationModal(toggleSubmitPopUp)"
+                [disabled]="!isExamOver() || isSaving"
+                [ngbTooltip]="!isExamOver() || isSaving ? ('artemisApp.studentExamDetail.disabledChangeSubmissionStateButtonExplanation' | artemisTranslate) : ''"
```

Comment:
Tooltips are not visible on disabled buttons, so this never displayed. One possible workaround might be to wrap the button into a div and have the tooltip on the div.

Reply:
Thank you for the review and the solution suggestion!

I should have tested it better; it is now adjusted :)